### PR TITLE
DOC: Add implemented tractography method table to tracking example index

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Fix grammar in local reconstruction example doc index passage
- DOC: Add implemented tractography method table to tracking example index

### CODE REVIEW

File: doc/tractography_methods_list.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     - :cite:t:`Garyfallidis2012b`
+   * - :ref:`Local probabilistic <sphx_glr_examples_built_fiber_tracking_tracking_probabilistic.py>`
+     - Local
+     - Probabilistic
+     - :cite:t:`Behrens2003`, :cite:t:`Behrens2007`
+   * - :ref:`Bootstrap tracking <sphx_glr_examples_built_fiber_tracking_tracking_bootstrap_peaks.py>`
+     - Local
+     - Probabilistic
+     - :cite:t:`Berman2008`
+   * - :ref:`PFT <sphx_glr_examples_built_fiber_tracking_tracking_pft.py>`
```
Comment (arokem) [inline doc/tractography_methods_list.rst:21]: Maybe spell out "Particle Filtering Tractography"?

File: doc/tractography_methods_list.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+     - Local
+     - Deterministic/Probabilistic
+     - :cite:t:`Girard2014`
+   * - :ref:`PTT <sphx_glr_examples_built_fiber_tracking_tracking_ptt.py>`
```
Comment (arokem) [inline doc/tractography_methods_list.rst:25]: Same here, maybe spell out the acronym?

File: doc/tractography_methods_list.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.. list-table:: Fiber tracking methods available in DIPY.
```
Comment (arokem) [inline doc/tractography_methods_list.rst:1]:
```suggestion
.. list-table:: Tractography methods available in DIPY.
```

**Review by arokem — COMMENTED** [review]
Looks great! A couple of small suggestions

File: doc/tractography_methods_list.rst (hunk 1 above)
Comment (jhlegarreta) [reply doc/tractography_methods_list.rst]: Done in 44c1deb.

File: doc/tractography_methods_list.rst (hunk 2 above)
Comment (jhlegarreta) [reply doc/tractography_methods_list.rst]: Done in 44c1deb.

File: doc/tractography_methods_list.rst (hunk 3 above)
Comment (jhlegarreta) [reply doc/tractography_methods_list.rst]: Changed in 44c1deb.

**Review by skoudoro — APPROVED** [review]
Thank you @jhlegarreta,

image

Looks good to me

**Pull request merged**